src/main/java/com/shrralis/ssdemo1/repository/RecoveryTokensRepository.java

```diff
@@ -17,4 +18,9 @@
 	Optional<RecoveryToken> findByToken(String token);
 
 	RecoveryToken getByToken(String token);
+
+	@Query(value = "SELECT COUNT(*) FROM " + RecoveryToken.TABLE_NAME +
+			" WHERE " + RecoveryToken.USER_COLUMN_NAME + " = ?1 AND " +
+			RecoveryToken.EXPIRES_AT_COLUMN_NAME + " >= NOW() ", nativeQuery = true)
```

Comment:
Please, don't be afraid of this

Reply:
There are no need to use constants. We are in repository for that entity so it's ok to just hardcode it

Reply:
O.K., I'll replace it with just a string query in future